src/Microsoft.DotNet.ProjectJsonMigration/Rules/TemporaryMutateProjectJsonRule.cs

```diff
@@ -39,8 +40,8 @@ public void Apply(MigrationSettings migrationSettings, MigrationRuleInputs migra
             }
 
             var json = CreateDestinationProjectFile(sourceProjectFile, destinationProjectFile);
-            InjectSdkReference(json, ConstantPackageNames.CSdkPackageName, migrationSettings.SdkPackageVersion);
             RemoveRuntimesNode(json);
+            RemoveAllDependencies(json);
```

Comment:
That's a good point, I was thinking that we still need it for tools, but that is not going to work